CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## Supported Release 1.6.0
+### Summary
+
+A substantial release made to create a clean base from which Rubocop may be implemented.
+#### Features
+- Mocha version updated.
+- Test's updated to match ruby version 2.0.0.
+- fast_gettext added to gems.
+- Locales folder and config.yml added.
+- Support added for balancemember weights.
+- Concat validate_cmd can be configured.
+- Space now added to headers for formatting.
+- Haproxy Revolver dded, only supported by Haproxy version 1.6+.
+- Update to match Puppet 4 datatypes.
+
+#### Bugfixes
+- Multiple Modulesync updates.
+- A listen check was added to the code.
+- System service dlap detection avoided during acceptance tests.
+- Undefined values have been dropped from config template.
+- Work around for usage of 'which' in unbuntu 12.04.
+- Verifyhost parameter added to balancemember resource.
+- Fix to $bind_options.
+- Fix to example ports listening value.
+- Fix to lint warnings.
+- validate_* replaced with datatypes.
```

Comment:
Is this puppet4 implementation? If so its backwards incompatible and would need an x release.